wbooks/src/app/screens/bookList/index.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { FlatList, ListRenderItem, SafeAreaView, View } from 'react-native';
+import { FlatList, ListRenderItem, SafeAreaView, View, TouchableOpacity, Text } from 'react-native';
 import { BOOKS_MOCK } from '@constants/mockBooks';
 import Book from '@components/book';
 import { Book as IBook } from '@interfaces/bookInterface';
 
 import styles from './styles';
 
-function BookList() {
+function BookList({ navigation }: any) {
```

Comment:
export the following code into a navigation interface and then use it here.

```import { RouteProp, NavigationProp, NavigationState } from '@react-navigation/native';

export interface Navigation {
  route: RouteProp<Record<string, object | undefined>, string>;
  navigation: NavigationProp<Record<string, object | undefined>, string, NavigationState, {}, {}>;
}
```

Reply:
the navigation interface was created